usermatch/src/schema.js

```diff
@@ -0,0 +1,21 @@
+const userSchema = new mongoose.Schema({
```

Comment:
Since this now lives in a different file, we need to export it.
`export const userSchema`